### PR TITLE
Add documentation for Azure Service Bus Configuration

### PR DESCRIPTION
**Pull request by preardon**

### CODE REVIEW

**Comment by preardon** [comment]
@iancooper let me know what you think

File: contents/AzureServiceBusConfiguration.md [hunk 1]
```diff
@@ -1,3 +1,121 @@
...
+
+* **ServiceBusConnectionStringClientProvider**: A client provider that accepts a connection string (containg Authentication information)
+
+* **ServiceBusDefaultAzureClientProvider**: A client provider that uses the Default Azure Credential to authenticate.
+
+* **ServiceBusManagedIdentityClientProvider**: A client provider that uses Azure Managed Identity to authenticate.
+
+* **ServiceBusVisualStudioCredentialClientProvider**: A client provider that uses Visual Studio Credential to authenticate.
+
+In Brighter's implimentation of the Messaging Gateway Publications and Subscvriptions have their own Individual configuration.
```
Comment (iancooper) [inline contents/AzureServiceBusConfiguration.md]: spelling, and italicize concepts like Subscription or Publication

Comment (iancooper) [reply contents/AzureServiceBusConfiguration.md]: OK, looks like you do that elsewhere

**Review by iancooper — APPROVED** [review]

**Comment by iancooper** [comment]
@preardon I approved, it looks great, there is one minor typo

**Comment by preardon** [comment]
@iancooper Fixed, doesn't look like I have any permissions on this repo so I can't merge. Please merge if happy

**Comment by iancooper** [comment]
@preardon Oooh, I wonder why you don't have rights. Merged.

**Pull request merged**